src/core/opamStd.ml

```diff
@@ -1007,8 +1006,7 @@ module OpamSys = struct
         r
 
   let system () =
-    (* CSIDL_SYSTEM = 0x25 *)
-    OpamStubs.(shGetFolderPath 0x25 SHGFP_TYPE_CURRENT)
+    OpamStubs.getPathToSystem ()
```

Comment:
This was already (my) bad, but this ought to be lazy (given the number of times we can end up calling cygpath, etc. - this function is used the Cygwin-processing in `OpamProcess`), something like:

```ocaml
let system =
  let system = Lazy.from_fun OpamStubs.getPathToSystem in
  fun () -> Lazy.force system
```